src/LaunchDarkly.ServerSdk/LaunchDarkly.ServerSdk.csproj

```diff
@@ -13,7 +13,7 @@
     <OutputType>Library</OutputType>
     <PackageId>LaunchDarkly.ServerSdk</PackageId>
     <RootNamespace>LaunchDarkly.Sdk.Server</RootNamespace>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>10.0</LangVersion>
```

Comment:
The C# Language is not the same as the TargetFramework.  You can `record struct` on the following target frameworks:

* net20
* net35
* net451
* net452
* net46
* net461
* net462
* net47
* net471
* net472
* net48
* netcoreapp1.0
* netcoreapp1.1
* netcoreapp2.0
* netcoreapp2.1
* netcoreapp2.2
* netcoreapp3.0
* netcoreapp3.1
* netstandard1.0
* netstandard1.1
* netstandard1.2
* netstandard1.3
* netstandard1.4
* netstandard1.5
* netstandard1.6
* netstandard2.0
* netstandard2.1
* net5.0
* net6.0

Reply:
Output from `net20`:

![image](https://user-images.githubusercontent.com/12101120/152626262-7df12aee-87ef-4753-a1f8-2f8260e928b4.png)